src/api/app/assets/javascripts/webui/request.js

```diff
@@ -138,6 +138,10 @@ $(document).ready(function(){
   $('#changes-item').on('shown.bs.tab', function () {
     loadChanges();
   });
+
+  var shiftWindow = function() { scrollBy(0, -57) };
```

Comment:
We definitely a comment here for this, especially for the `-57`. What is this value exactly? The height of the navbar?

Reply:
Yes, the height is 57

Reply:
Hm, isn't this broken for every anchor on every page?

Reply:
Yes and with this approach, it solves the issue, but only partially. For example, if you click on an anchor, then scroll a bit to look at something else, then click on the anchor again, it's broken again:

![preview](https://user-images.githubusercontent.com/1102934/184652826-9bdb79c2-336e-410d-a330-1262460ce059.gif)

Reply:
I'm not sure if this is broken for every anchor. Do you have any examples where it is broken outside this page?

Reply:
Yes, in the comments on the project#show page. Same for the package#show page.